packages/react/src/elements/components/GlobalNav/SideNav/SideNav.js

```diff
@@ -80,9 +93,14 @@ class SideNav extends Component {
       higTheme: this.props.higTheme
     };
 
+    const SideNavVariant =
```

Comment:
You can have React pick between different component types at runtime. The important thing is for this variable to be capitalized.

https://reactjs.org/docs/jsx-in-depth.html#choosing-the-type-at-runtime

Reply:
I love it.